README.md

```diff
@@ -186,8 +186,16 @@ Default is 0 seconds. Maximum is 604,800 seconds (7 days).
 ### Delete a Message from a Queue
 
 ```javascript
-queue.del(message_id, function(error, body) {});
+queue.del(message_id, {}, function(error, body) {});
```

Comment:
I think it is better to create the 'options' variable and mention that it can contain reservation_id and subscriber_name if needed (will be clearer):
var options = {};
queue.del(message_id, options, function(error, body) {});